frontend/src/containers/Faq.js

```diff
@@ -96,6 +96,9 @@ export class Faq extends Component {
           <h3>Why do you have monthly payments as default?</h3>
           <p>A collective is an organization of people working together towards a common goal. Sometimes these goals take a few months and sometimes they can take years. By supporting a collective with a recurring monthly amount, you help the collective plan out its future activities.</p>
           
+          <h3 id='charge-date-notice'>Why do you charge recurring monthly subscriptions on 1st of every month?</h3>
+          <p>It helps our collectives manage their monthly budget. Instead of getting donations randomly throughout the month, they can start a month knowing a minimum available budget for that month. Any new donations during the month only improves their budget.</p>
+          
```

Comment:
What about also adding a paragraph about saying that it will also help us bundle together (in the future) donations made to multiple collectives and therefore save on transaction fees.